sentry-core/src/main/java/io/sentry/core/AsyncConnectionFactory.java

```diff
@@ -17,7 +17,7 @@ public static AsyncConnection create(
         options.getTransportGate(),
         eventCache,
         sessionCache,
-        options.getCacheDirSize(),
+        options.getMaxQueueSize(),
```

Comment:
M: MaxQueueSize indicates the number of items in-memory that we can hold before dropping. Is this what this is?
The original thing meant max items in the file system, right?

Reply:
yes, exactly, I believe with session + events, using the CacheDirSize doesn't make sense anymore and we should have a specific option (MaxQueueSize) for that